### PR TITLE
chore(images): split images into versioned subdirectories, update 2.2.0 images, and update various 2.2.0 guides

### LINKED ISSUE #74: [Task] Split images directory into versioned subdirectories
Images should have versioned subdirectories corresponding to the versioned guides.

### PR DESCRIPTION
**Pull request by maxcao13**
Fixes #74 
Related #45 
Related #67

### CODE REVIEW

**Comment by andrewazores** [comment]
General comment not specific to this PR, but something I noticed in reviewing it:

Under the "Archive a Recording" guide, there is the following phrase:

>  The active source recording may be continuous or fixed-duration, using any event template or event specifier string

"event specifier string"s haven't been supported for a while now so we should drop mentions. This looks like the only place it shows up.

**Comment by andrewazores** [comment]
Also under "Archive a Recording",

>  All recordings in the archives are displayed here, regardless of the selected target application. The name of the archived recording reflects the address of the target application and the original name of the active recording that it was retrieved from, and includes a timestamp indicating when the archived recording was created. 

This isn't true anymore, that table is powered by GraphQL and shows target-specific archives :-)

**Comment by andrewazores** [comment]
"View a Recording in Grafana" can be updated.

> Once logged in, navigate to the dashboard titled Dashboard.

User shouldn't need to navigate within Grafana anymore, it should jump right into the default dashboard.

Step 7 from this guide can be removed since it's detailing how to do this. Step 8 needs a new screenshot.

**Comment by andrewazores** [comment]
Looks great otherwise, thanks for doing all this work.

**Review by andrewazores — COMMENTED** [review]
LGTM, I'll wait to see if anyone else catches any issues.

File: guides/_subsections/automated-analysis.md [hunk 1]
```diff
@@ -54,25 +50,39 @@ for offline use.
...
+        </figcaption>
+      </figure>
+      <figure>
+        <a href="{{ site.url }}/images/2.2.0/automated-analysis-5.png" target="_blank">
+          <img src="{{ site.url }}/images/2.2.0/automated-analysis-5.png">
+        </a>
+        <figcaption>
+          To download the HTML <i>Automated Analysis</i> report to local disk, right click 
+          the page and select <i>Save Page As...</i>. Alternatively, press <kbd>Ctrl</kbd>
+          +<kbd>S</kbd> on Windows/Linux, or <kbd>⌘</kbd>+<kbd>S</kbd> on macOS.
```
Comment (tthvo) [inline guides/_subsections/automated-analysis.md:72]: The css works nice :D Probably remove the extra space after `Ctrl`? Or add one between all?

Screenshot from 2022-11-02 14-35-09

File: guides/_subsections/re-upload-a-recording-to-archives.md [hunk 2]
```diff
@@ -8,40 +8,43 @@ previously retrieved recording in a new Cryostat instance.
...
-      summary="Select the Recording to Upload"
-      image-name="re-upload-a-recording-to-archives-2.png"
+      summary="<i>(Optional)</i> Add metadata labels"
+      image-name="2.2.0/re-upload-a-recording-to-archives-3.png"
       caption="
-        Click the '+' button, then click <i>Browse...</i> and select the .jfr
-        file to upload. Then click 'Submit'.
+        Click <i>Show metadata options</i> to add optional metadata labels to recording.
+        You can either add labels by clicking the <i>Add Label</i> button, or by uploading
+        a custom <code>.json</code> labels file from your local storage. For more on Cryostat
```
Comment (tthvo) [inline guides/_subsections/re-upload-a-recording-to-archives.md:37]: Should we add a sample of this `.json` file or mention it is the same as the one saved when downloading a recording?

**Review by tthvo — COMMENTED** [review]
Looks good! Just one final question :D

File: guides/_subsections/re-upload-a-recording-to-archives.md (hunk 2 above)
Comment (andrewazores) [reply guides/_subsections/re-upload-a-recording-to-archives.md:37]: I think just mentioning that it's the same one that would be downloaded is good. It's somewhat subject to change between versions, and I don't expect users will all that often be manually editing these files anyway. Much more likely to just select the accompanying metadata download to restore it.

**Review by tthvo — APPROVED** [review]
Looks good to me :D

**Pull request merged**